lib/ts/recipe/passwordless/utils.ts

```diff
@@ -158,3 +159,28 @@ export function defaultGuessInternationPhoneNumberFromInputPhoneNumber(
     }
     return value;
 }
+
+export async function getLoginAttemptInfoFromStorage(input: {
```

Comment:
Is this util here just to help with types?

Reply:
Yes and to prevent the need to pass a generic every time we call `getLoginAtemptInfo` on the recipe interface (also refer to the comment before this oen)

Reply:
Well the idea was to maintain the recipe function name in the util function name to make it clear that they are co-related (`getLoginAttemptInfo` being the recipe function). There is an argument that I could have used the same name and done it that way but I think that might be confusing

Reply:
I think the same name might be fine, or you could add `ForReact` as a suffix because I think FromStorage is not 100% correct: it indicates local (or session) storage but it could just as well be customized. 
Anyway, this is fine as it is if you prefer it this way.